themes/custom/manatime_theme/gulpfile.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
```

Comment:
Delete this file. Instead, copy gulp-tasks/theme.js from retopais to root gulp-tasks folder and change appropiate values.